enkit/outputs/commands.go

```diff
@@ -64,13 +67,6 @@ func NewRoot(base *client.BaseFlags) (*Root, error) {
 		BaseFlags: base,
 	}
 
-	homeDir, err := os.UserHomeDir()
-	if err != nil {
-		return nil, fmt.Errorf("failed to detect $HOME: %w", err)
-	}
-	defaultOutputsRoot := filepath.Join(homeDir, "outputs")
-
-	rc.PersistentFlags().StringVar(&rc.OutputsRoot, "outputs-root", defaultOutputsRoot, "Root dir of mounted outputs")
```

Comment:
`enkit` is no longer responsible for creating the mount so there is no need to make it user configurable